contracts/src/sample_erc20.cairo

```diff
@@ -0,0 +1,253 @@
+use starknet::ContractAddress;
+
+#[starknet::interface]
+trait IERC20<TContractState> {
```

Comment:
use Open Zeppelin ERC20 component. You can find an example in their readme here: https://github.com/OpenZeppelin/cairo-contracts?tab=readme-ov-file#using-the-library